staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3180,6 +3196,14 @@ type PodSecurityContext struct {
 	// sysctls (by the container runtime) might fail to launch.
 	// +optional
 	Sysctls []Sysctl `json:"sysctls,omitempty" protobuf:"bytes,7,rep,name=sysctls"`
+	// FSGroupChangePolicy defines behavior of changing ownership and permission of the volume
```

Comment:
Your godoc on external types must use the name of the field as it would be seen in json, i.e. `fsGroupChangePolicy defines ...`

Reply:
fixed.